librocksdb-sys/build.rs

```diff
@@ -187,6 +188,10 @@ fn build_snappy() {
         config.flag("-std=c++11");
     }
 
+    if endianness == "big" {
+        config.define("WORDS_BIGENDIAN", Some("1"));
```

Comment:
I couldn't find a usage of the `WORDS_BIGENDIAN` var. Is it really correct name ?

Reply:
I think you're right, good catch. @knarz should it be `SNAPPY_IS_BIG_ENDIAN`?

Reply:
The snappy version pulled in here is over two years old and has `#ifdef WORDS_BIGENDIAN` as the only only reference to endianness.
Your suggestion seems to be correct for a current version of snappy

Reply:
Yes. In a revision we use really presents `WORDS_BIGENDIAN`. So. Maybe update snappy for fresher revision like [this](https://github.com/google/snappy/commit/537f4ad6240e586970fe554614542e9717df7902) for using an actual var `SNAPPY_IS_BIG_ENDIAN` ?

Reply:
It's really up to you. Actually, I can merge it as is, but I think to update snappy revision in any case even in other PR.

Reply:
I'd say merge this as is and update snappy later.

Updating snappy would definitely be in order, seeing as the version used is very old. Unfortunately I don't have the time to work on that at the moment—I assume it's a bigger task.